app/views/application/_sidebar.html.erb

```diff
@@ -14,6 +14,14 @@
           <span class="ml-4">Mentoring</span>
         <% end %>
       </li>
+      <li class="relative px-6 py-3">
+        <%= link_to new_admin_allocation_chart_index_path, class: "inline-flex items-center w-full text-sm font-semibold transition-colors duration-150 hover:text-gray-800 dark:hover:text-gray-200" do %>
+          <svg class="w-6 h-6" data-darkreader-inline-stroke="" fill="none" stroke="currentColor" stroke-width="1.5" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" aria-hidden="true">
+            <path stroke-linecap="round" stroke-linejoin="round" d="M7.5 14.25v2.25m3-4.5v4.5m3-6.75v6.75m3-9v9M6 20.25h12A2.25 2.25 0 0020.25 18V6A2.25 2.25 0 0018 3.75H6A2.25 2.25 0 003.75 6v12A2.25 2.25 0 006 20.25z"></path>
+          </svg>
```

Comment:
@adbatista just checking with you first... would not be better to save this `<svg>` in a file `.svg` and use it like:

```ruby
 <%= image_tag '[image-name].svg', class: 'w-6 h-6' %>
```
?

Reply:
🤔 Yeah makes sense, this way we keep things separated

Reply:
@samkevin1 can you do that?

Reply:
sure, i'll change it

Reply:
in which folder should I save these `.svg`s?

Reply:
app/assets/images

Reply:
when I use `image_tag` it becomes a <img> tag and I can't change the color of the icon, any suggestions of what I can do to fix this?

Reply:
Well... I think you have 3 options:

A) Create our own helper to render the SVG as `<svg />`: https://stackoverflow.com/a/36992492/4538055
B) Use this gem: [inline_svg](https://github.com/jamesmartin/inline_svg)
C) Keep it the way it is.

I didn't realize `image_tag` would create a `<img />` tag. I prefer option `B` over option `A` just to make sure it works  but maybe we don't want another gem and option `A` can get complicated (maybe)

So, we would go with option `C`.

Maybe @adbatista can help us with that question.

Reply:
Forgot to comment, but I changed this. I'm saving the svg's in `application/images` and rendering it on the page. Can you take a look again?

Reply:
Changed it, it's using `inline_svg` now @adbatista @nolleto